.github/workflows/auto_community_labeler.yml

```
@@ -0,0 +1,28 @@
+
+name: Auto Community Labeler
+
+on:
+  pull_request:
+  issues:
+
+jobs:
+  auto_PR_labeler:
+    name: Auto PR Community Labeler
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' && github.event.pull_request.author_association != 'MEMBER' && github.event.pull_request.author_association != 'OWNER'
+
+    steps:
+      - name: Add Contributer Label
```

There's a typo `Contributer` -> `Contributor`
```suggestion
      - name: Add Contributor Label
```

I'm wondering however whether "Add Community Contribution Label" wouldn't a clearer name. What do you think?